fem/fe.hpp

```diff
@@ -1712,7 +1713,7 @@ class Poly_1D
    QuadratureFunctions1D quad_func;
 
 public:
-   Poly_1D() { }
+   Poly_1D(): h_mt(Device::GetHostMemoryType()) { }
```

Comment:
I think we can always use `MemoryType::HOST` here -- this type is used only for allocating `Array`s of pointers. This way, the `h_mt` variable can be simply removed (or made static const).

Reply:
`done`